src/main/java/PasswordGenerator.java

```diff
@@ -1,8 +1,74 @@
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import java.util.Random;
+
+import static java.lang.Math.random;
+
 /*
  * This Java source file was generated by the Gradle 'init' task.
  */
 public class PasswordGenerator {
-    public String generatePassword() {
-        return "a+1";
+
+    List myPassword = new ArrayList<>();
```

Comment:
The lack of type causes warnings further down about unchecked operations. This should probably be a `List<Character>`.